.github/workflows/ci-cd-pull-request.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           registry: ghcr.io
           username: Jannis
```

Comment:
Do we have to update this as well @nenadjaja?

Reply:
good question. I should update it to my account

Reply:
Should it be another "secret" (environment variable), since it might change?